roofit/roofit/inc/RooBernstein.h

```diff
@@ -33,11 +33,13 @@ class RooBernstein : public RooAbsPdf {
 
   Int_t getAnalyticalIntegral(RooArgSet& allVars, RooArgSet& analVars, const char* rangeName=0) const ;
   Double_t analyticalIntegral(Int_t code, const char* rangeName=0) const ;
+  virtual void selectNormalizationRange(const char* rangeName=0, Bool_t force=kFALSE) ;
```

Comment:
```suggestion
void selectNormalizationRange(const char* rangeName=0, Bool_t force=kFALSE) ;
```
Let's make it virtual only when needed, i.e., when somebody decides to derive from RooBernstein.